cyfronet-fid/sat4envi
# [#794] Preserve query params (sceneId & date & overlays)

##What is in this PR?

* refactor map-view.component.ts to handle asynchronous tasks in a better way

## How to test it

1. Configure map (select products, overlays, view)
2. Copy URL
3. paste url in new window
4. the same map state as in other window should be set.

Closes: #794
